docs/src/main/asciidoc/security-built-in-authentication.adoc

```diff
@@ -7,17 +7,46 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 
 include::./attributes.adoc[]
 
-This document describes the Quarkus built-in authentication mechanisms for HTTP based FORM, BASIC and Mutual TLS authentication as well as the proactive authentication.
+The following section describes the Quarkus built-in authentication mechanisms for HTTP based FORM, BASIC, and Mutual TLS authentication. Proactive authentication is also described.
```

Comment:
```suggestion
Explore Quarkus built-in authentication mechanisms and behaviors. Quarkus supports HTTP FORM, BASIC, and Mutual TLS authentication and will proactively authenticate users by default.
```

Do we need words like "the following section" in here at all? In most cases, that scope is clearly implied, making it redundant. This should be a hook: why do I want to read this? The typical (in my past) word for that was "Learn.." but diataxis has a good justification for why that isn't the best choice.

Abstracts are hard, but I would like to use these for the tiles on the website at some point (automagically to reduce manual fiddling), which requires being concise (another reason to remove the "In this document.. " phrasing).